### ISSUE TITLE
Implement DKQ element

### ISSUE THREAD

**Issue by JHenneberg** [JuliaFEM/FEMPlates.jl#12]
DKQ - Discrete Kirchhof Quadrilateral
The DKT element is already implemented and the DKQ element does not really need a lot of adjustments.

**Comment by ahojukka5** [comment]
Yes, you are right on this. There's still a couple of issues I would like to consider before continuing on plate and shell development. The first is that we should be able to couple elements with continuum elements, and also rotate elements in 3d space, to make some practical use for these in real-life simulations. Another thing I would like to consider is how the basis functions, which depends on the geometry of the elements, should be defined.

**Comment by JHenneberg** [comment]
Sure I understand. Anyway I need to dive deeper into Julia and this project Looks pretty interesting so far, because otherwise there only pretty much legacy open source programs for FEM.

**Comment by ahojukka5** [comment]
@ahniemi is our plate & shell expert if you need some theoretical guidance. And of course, you can always ask from me if there are some parts of code hard to understand.

**Comment by JHenneberg** [comment]
Sounds good. Is there any roadmap which elements you want to implement?

**Comment by ahojukka5** [comment]
Well, I'm working on assumed strain 8-node solid-shell element.
